.gitignore

```diff
@@ -43,3 +43,6 @@ coverage-report.xml
 qa/
 dist/
 .env
+
+# Ignore poetry lock file
+poetry.lock
```

Comment:
We should not do this. This will make the developers environments, as well as CI/CD pipelines non deterministic: https://python-poetry.org/docs/basic-usage/#committing-your-poetrylock-file-to-version-control

Reply:
Yes, indeed, the poetry.lock should include everything, including optional dependencies.